nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/PutElasticsearch.java

```diff
@@ -99,6 +99,14 @@
                     AttributeExpression.ResultType.STRING, true))
             .build();
 
+    public static final PropertyDescriptor INDEX_OP = new PropertyDescriptor.Builder()
+            .name("Index Operation")
+            .description("The type of the operation used to index (index, update, upsert)")
```

Comment:
The description mentions three modes but only two are allowed. Should they be "insert" and "update/upsert", or is "index" the preferred terminology for adding a new document?